src/js/components/tablero/tablero.scss

```diff
@@ -26,91 +26,107 @@
   .numberCardComponent {
     padding-left: 4%;
   }
+}
+
+.cardComponent {
```

Comment:
I am not sure how this slipped in, but we are using kebab case naming convention for css classes. I see that you used camel case in whole file, i guess we have to leave it.

Reply:
We can change everything to kebab case, it shouldn't be too complicated

Reply:
Cool, i think we should have the same code style (at lest on the react side of the project)